.github/workflows/_build_artifacts.yml

```diff
@@ -132,6 +132,16 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        # Exclude debug builds for non-amd64 targets since they won't be used,
+        # and trying to nodejs inside cross-platform containers has issues.
+        # See https://github.com/nodejs/docker-node/issues/1946
+        # Syntax is weird because https://github.com/actions/runner/issues/1512
+        exclude:
```

Comment:
We also unnecessarily build `snownet-tests` and `http-test-server` for the perf tests. Those are only needed as debug builds for the integration tests.

Can we exclude those too?

Reply:
Done